source/reference/options.txt

```diff
@@ -386,21 +386,11 @@ Authentication Options
 
 .. option:: --authenticationMechanism <name>
 
-   *Default*: SCRAM-SHA-1
+   Default: SCRAM-SHA-256
```

Comment:
This isn’t really accurate. Matthew’s comment in the JIRA ticket is correct: The default (if credentials were specified), is to try _both_ SCRAM-SHA-256 and SCRAM-SHA-1, so specifying no `--authenticationMechanism` _is_ different from specifying `--authenticationMechanism SCRAM-SHA-256`. I think it would be misleading to say that `SCRAM-SHA-256` is the default value.

(If no auth mechanism is explicitly specified, but credentials are provided, mongosh (and drivers) will prefer SCRAM-SHA-256 if available, then fall back to SCRAM-SHA-1.)